### ISSUE TITLE
Adding Job Requirements (YAML) with matchType Exists Fails

### ISSUE THREAD

**Issue by mayani** [mibexsoftware/bamboo-plan-dsl-plugin#38]
### Add On Version: 
2.22.3

### Sample Input (Job Requirement)

```yaml
requirements:
  - capabilityKey: system.builder.command.docker
    matchType: !exists exists
```

### Error Message

```
Invalid YAML syntax: Can't construct a java object for !exists; exception=No single argument constructor found for class ch.mibex.bamboo.plandsl.dsl.jobs.Requirement$Exists in 'string', line 21, column 30: matchType: !exists ^
```

**Comment by mrueegg** [comment]
Hi,

Thanks for the detailed bug report! 

I can confirm this issue. Unfortunately, there is no workaround. We will fix it in the next plug-in release coming out within 1-2 days. I'll report back here when it is available.

Best regards,
Michael

**Comment by mrueegg** [comment]
Fixed with release 1.8.2. Thanks again for reporting this!

Best regards,
Michael
